presto-postgresql/src/main/java/com/facebook/presto/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -57,12 +57,12 @@ public void commitCreateTable(JdbcOutputTableHandle handle, Collection<Slice> fr
     }
 
     @Override
-    public Statement getStatement(Connection connection)
+    public PreparedStatement getPreparedStatement(Connection connection, String sql)
             throws SQLException
     {
         connection.setAutoCommit(false);
-        Statement statement = connection.createStatement();
```

Comment:
Nit: we can keep the variable name as `statement` to reduce the number of changed lines (either one is a good name, so might as well stick the the shorter one)